frontend/src/metabase/visualizations/components/CardRenderer.jsx

```diff
@@ -69,15 +72,24 @@ export default class CardRenderer extends Component {
     this._deregisterChart();
 
     // reset the DOM:
-    let element = parent.firstChild;
-    if (element) {
-      parent.removeChild(element);
+    for (const child of parent.children) {
+      parent.removeChild(child);
     }
 
     // create a new container element
-    element = document.createElement("div");
+    const element = document.createElement("div");
     parent.appendChild(element);
 
+    if (isDashboard && isEditing && !isSettings) {
+      // If this card is a dashboard that's currently being edited, we cover the
+      // content to prevent interaction with the chart. The !isSettings
+      // exception is to handle modals that appear above a dashboard.
```

Comment:
Instead of creating an extra div could you just set `pointer-events` to `none` on `element` itself, or does that not work because some child elements have `pointer-events` set on them?

Reply:
Yeah that doesn't work because the children override it. `g.brush` in the chart has `pointer-events: all` set.